app/services/container_dashboard_service.rb

```diff
@@ -20,8 +20,8 @@ def all_data
   end
 
   def status
-    if @ems.present? && @ems.kind_of?(ManageIQ::Providers::Openshift::ContainerManager)
-      routes_count = @ems.container_routes.count
+    if @ems.present?
+      routes_count = @ems.respond_to?(:container_routes) ? @ems.container_routes.count : 0 # ems might not have routes
```

Comment:
not a big fan of the "try", I prefer the respond_to, but the above is also a legit option